faults.csv

```diff
@@ -1,63 +1,70 @@
+PPS,Not Ready for Beam Chain A,PPS: West Global not in Ready for Beam,Communicate the fault to the EOIC and await resolution,ALL,,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyA,1,,2,Not Ready for Beam,
+PPS,Not Ready for Beam Chain B,PPS: West Global not in Ready for Beam,Communicate the fault to the EOIC and await resolution,ALL,,EDM,$EDM/pps/pps_linacwest.edl,,,PPS:SYSW:1:,BeamReadyB,1,,2,Not Ready for Beam,
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a. rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Verify that the rack is not actively being worked on and run Rack Hardware Init and Test > Rack Checkout > Go,RACK,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,Hardware Initialization needed,
+HWI,Hardware Initialization needed,Hardware Initialization (a.k.a rack checkout) needs to be run or other LLRF chassis problem. Don't trust current values.,Verify that the rack is not actively being worked on and run Rack Hardware Init and Test > Rack Checkout > Go,RACK,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,2,2,Hardware Initialization needed,
+HWR,Hardware Initialization running,Hardware Initialization (ak.a. rack checkout) currently running. Don't try to use system until this is complete,Wait for fault to clear,RACK,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",A,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,1,1,Hardware Initialization running,
+HWR,Hardware Initialization running,Hardware Initialization (a.k.a. rack checkout) currently running. Don't try to use system until this is complete,Wait for fault to clear,RACK,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=8,SELCHAR=3",B,ACCL:{LINAC}:{CRYOMODULE}00:RACK{RACK}:,HWINITSUM,,1,1,Hardware Initialization running,
+NRP,NIRP or individual SSA interlock,NIRP or other individual SSA interlocks like PPS,"If the realtime status is OK (check legend), reset interlocks. If not, contact NIRP system experts.",SSA,,PYDM,$PYDM/nirp/nirp_Lband_diagnostic_graphical.ui,"{""AREA"": {LINAC}, ""CM"": {CRYOMODULE}, ""RFNAME"": ""CM{CRYOMODULE}""}",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,NRP_PRMT,1,,2,NIRP or individual SSA interlock,
+SSP,SSA Permit,SSA permit to turn on faulted,Run auto setup,CAV,,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,SSA_LTCH,0,,2,SSA Permit,
+SSA,SSA Faulted,SSA not on,Run auto setup,SSA,,EDM,$EDM/llrf/rf_srf_ssa_{cm_OR_hl}.edl,,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:SSA:,FaultSummary.SEVR,,2,2,SSA Faulted,
+BCS,BCS LLRF Drive Fault,BCS fault is interrupting LLRF drive (only affects CM01 in practice),Communicate the fault to the EOIC and await resolution,CM,ALL,EDM,$EDM/bcs/ops_lcls2_bcs_main.edl,,,ACCL:{LINAC}:{CRYOMODULE}00:,BCSDRVSUM,0,,2,BCS LLRF Drive Fault,

```

Comment:
What does the "ALL" in the 'CM Type' column do? Is that just place holder text so that the text box is not empty for a fault of level CM?

Reply:
- 1.3 GHz 
- 3.9 GHz
- All (both)

Does that make sense?

Reply:
Ah yep, that makes sense!